### PR TITLE
dead space 2

### PR DESCRIPTION
**Pull request by r3sus**

### CODE REVIEW

**Comment by FransBouma** [comment]
The deadspace 2 guide doesn't have a header I think? 

You refer to a link via webarchive but I think that link doesnt work anymore, or is it still active?

**Comment by r3sus** [comment]
Yes, there's no header, I might add it later, or will, if it's required.
The page is archived and accessible via provided link.

**Comment by FransBouma** [comment]
Please provide a header image in line with the rest of the guides, thanks

**Comment by originalnicodr** [comment]
Oh look at that, thats one of jim shots!
Anyway with some upscaling IA on the original pic I managed to make a banner with the style we use on the guides :)
dead space 2-4

Thanks for collaborating to the site!

File: markdown/GameGuides/DeadSpace.md [hunk 1]
```diff
@@ -55,13 +55,13 @@ Based on [original description by Jim2Point0](https://web.archive.org/web/201410
 
 FOV: Field of View.
 
-Cam Distance: How far back the camera sits from character. The further you go, the smaller he is. This is what the "widescreen fixer" app modifies as well.
+Cam Distance: How far back the camera sits from character. The further you go, the smaller char is. This is what the "widescreen fixer" app modifies as well.
```
Comment (FransBouma) [inline markdown/GameGuides/DeadSpace.md:58]: Please next time, submit changes to different guides in different PRs. Also this change is silly, what's 'char' ?

Comment (r3sus) [reply markdown/GameGuides/DeadSpace.md:58]: Ok, got it.
Char is short for Character. But for sure 'he' would fit here. 
Thanks.
